themes/openy_themes/openy_carnation/openy_carnation.theme

```diff
@@ -265,4 +458,111 @@ function openy_carnation_form_system_theme_settings_alter(&$form, FormStateInter
+    '#open' => TRUE,
+  ];
+  $form['camp_section']['openy_carnation_camp_section_logo'] = [
+    '#type' => 'managed_file',
+    '#title' => t('Camp section logo'),
+    '#description' => t('To display a logo in the header area on camp pages, and camp related landing pages, upload it here.</br>Image size suggested 92 x 70 pixels.'),
+    '#upload_validators' => [
+      'file_validate_extensions' => ['png jpg jpeg gif svg'],
+    ],
+    '#upload_location' => 'public://',
+    '#default_value' => theme_get_setting('openy_carnation_camp_section_logo'),
+  ];
+  $form['camp_section']['openy_carnation_camp_favicon'] = [
+    '#type' => 'managed_file',
+    '#title' => t('Camp favicon'),
+    '#description' => t('To display a favicon on camp pages, and camp related landing pages, upload it here.</br>Image sizes must be 16×16, 32×32, 48×48, or 64×64 pixels.</br>Accepted file types ico, png, gif.'),
+    '#upload_validators' => [
+      'file_validate_extensions' => ['ico png gif'],
+    ],
+    '#upload_location' => 'public://',
+    '#default_value' => theme_get_setting('openy_carnation_camp_favicon'),
+  ];
+
+  // Submission handler is not working on theme form
+  // See https://www.drupal.org/node/1862892 & https://www.drupal.org/node/2779947
+  // Setting file usage for openy_carnation theme.
+  if (($file = openy_carnation_entity_file_by_fid(theme_get_setting('openy_carnation_camp_section_logo'))) !== NULL) {
+    openy_carnation_set_theme_file_usage($file);
+  }
+  if (($file = openy_carnation_entity_file_by_fid(theme_get_setting('openy_carnation_camp_favicon'))) !== NULL) {
+    openy_carnation_set_theme_file_usage($file);
+  }
+}
+
+function openy_carnation_matchheight_settings(array &$form, FormStateInterface &$form_state) {
+  $form['matchheight_section'] = array(
+    '#type' => 'details',
+    '#title' => t('jQuery MatchHeight'),
+    '#description' => t('MatchHeight makes the height of all selected elements exactly equal.</br><a href="https://brm.io/jquery-match-height/">https://brm.io/jquery-match-height/</a></br>While this used to be a tricky problem, its advisable to use CSS Flexbox and</br> CSS Grid where possible these days, though this library may still be useful for legacy browsers.'),
+    '#open' => TRUE,
+  );
+  $form['matchheight_section']['matchheight_css_classes'] = array(
+    '#description' => t('CSS selectors for matchheight library'),
+    '#type' => 'textarea',
+    '#rows' => '10',
+    '#cols' => '10',
+    '#resizable' => 'vertical',
+    '#default_value' => theme_get_setting('matchheight_css_classes')
+  );
+}
+
+/**
+ * @param $fid_array
+ *
+ * @return \Drupal\file\Entity\File|null
+ */
+function openy_carnation_entity_file_by_fid($fid_array) {
+  if (!empty($fid_array) && is_array($fid_array) && is_numeric(reset($fid_array))) {
+    $file = File::load(reset($fid_array));
+    if ($file !== NULL) {
+      return $file;
+    }
+  }
+
+  return NULL;
+}
+
+/**
+ * @param $fid_array
```

Comment:
same here
try php sniffer with Drupal codestyle or use IDE for highlighting wrong docblocks

Reply:
Updated